shared_model/validators/field_validator.hpp

```diff
@@ -29,17 +29,21 @@
 namespace shared_model {
   namespace validation {
 
+    constexpr auto default_future_gap =
```

Comment:
Can you add documentation to this constant to explain what role it plays since it is a public constant?